### DISCUSSION TITLE
auto-enable aa

### DISCUSSION THREAD

**Discussion by svnomore** [Fedoraware/Fedoraware#1063] [Ideas]
auto-enable aa if in sniper's line of sight or in local fov

**Comment by canedoly** [comment]
or bind the aa to a key, and just press the key if you're in danger

**Reply by RealMuffinmaker** [reply]
how does bro with his slow aah brain know when he is in danger. go give him his aa on danger >:(

**Comment by Umbrozium** [comment]
fakelag on key exists, and fakelag on visibility is also an option. i see no issue in making the same thing for anti-aim.

**Reply by canedoly** [reply]
feel free to add it yourself, (the cheat is open source), you could use some code from fakelag on visibility + vaccinator player threat code to get something near aa on danger. But a key and having basic awareness is also a good choice
